app/src/main/res/layout-land/app_version_fragment.xml

```diff
@@ -15,34 +15,27 @@
 
     <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
-      android:layout_height="match_parent">
+      android:layout_height="match_parent"
+      android:background="@color/profileEditBackground">
 
       <TextView
         android:id="@+id/app_version_text_view"
         android:layout_width="match_parent"
-        android:layout_height="56dp"
+        android:layout_height="wrap_content"
         android:background="@color/white"
         android:fontFamily="sans-serif"
-        android:gravity="center_vertical"
         android:paddingStart="36dp"
+        android:paddingTop="20dp"
+        android:paddingEnd="36dp"
+        android:paddingBottom="20dp"
+        android:shadowColor="@color/oppiaTextShadow"
```

Comment:
This is incorrect, this does not work on all android versions.
Try using something from this link: https://stackoverflow.com/questions/15333529/how-to-provide-shadow-to-button

Do this in landscape as well as portrait.

Reply:
@rt4914 
thankyou modified accordingly